jina/serve/runtimes/gateway/__init__.py

```diff
@@ -28,3 +30,54 @@ def __init__(
         if self.timeout_send:
             self.timeout_send /= 1e3  # convert ms to seconds
         super().__init__(args, cancel_event, **kwargs)
+
+    @classmethod
```

Comment:
staticmethods cannot be inherited?

Reply:
I changed this, but wait_for_ready_or_shutdown has to stay classmethod